QuadPaySDK/PaymentWidget/PaymentWidget.swift

```diff
@@ -159,20 +163,21 @@ extension PaymentWidget {
                     self.timelapseGraphView.maxFee = self.maxFee
                     
                     
-          
                     DispatchQueue.main.async {
                         self.layout()
                     }
                     
                 case .failure(let error):
                     print(error)
+                    self.maxFee = 0
                     DispatchQueue.main.async {
                         self.layout()
                     }
                     
                 }
             }
```

Comment:
Should there be a return statement here?

Reply:
I dont think return statement is actually needed as we want to proceed on the dispatchQueue to unblock the thread so adding a return before line 172 will cause issues.